modules/core/src/main/scala/iota/Cop.scala

```diff
@@ -68,6 +68,12 @@ object Cop {
       else None
     def apply(a: A): Cop[L] = inj(a)
     def unapply(c: Cop[L]): Option[A] = proj(c)
+
+    def fold(c: Cop[L]): Either[Cop[TList.Op.Remove[A, L]], A] =
```

Comment:
Oops, didn't rename this to `projEither` (like in `CopK.InjectL`).
I don't really like the name `projEither` though, any suggestions for a better name?